docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md

```diff
@@ -436,7 +436,11 @@ espresso:
 ### `app`
 <p><small>| REQUIRED | STRING |</small></p>
 
-The path to the app. The default directory is `{project-root}/apps/filename.apk`, and the property supports expanded environment variables to designate the path, as shown in the following examples, or an already uploaded app reference. Supports \*.apk and \*.aab files.
+Either a local path, url, or storage identifier to the app under test. This property supports expanded environment variables.
+
+When defining a local path, the default directory is `{project-root}/apps/filename.apk`. The app will be uploaded to the Sauce Labs storage service. Supports \*.apk and \*.aab files.
+
+When defining a url to your app, it will be downloaded to a local temporary directory before being uploaded to the storage service.
```

Comment:
```suggestion
Specifies a local path, URL, or storage identifier to the app under test. This property supports expanded environment variables.

When defining a local path, the default directory is `{project-root}/apps/filename.apk`. The app will be uploaded to the Sauce Labs storage service. Supports \*.apk and \*.aab files.

When defining a URL to your app, it will be downloaded to a local temporary directory before being uploaded to Sauce storage.
```